TEMPLATE/etc/sssd/sssd.conf.erb

```diff
@@ -0,0 +1,35 @@
+[domain/default]
+
+autofs_provider = ldap
+ldap_schema = rfc2307bis
+ldap_search_base = <%= ldapbasedn %>
+id_provider = ldap
+auth_provider = ldap
+chpass_provider = ldap
+ldap_uri = <%= ldapserver %>
```

Comment:
I wonder if these two should match the parameters they replace,  ldapbasedn -> ldap_search_base and ldapserver -> ldap_uri for consistency (and doc clarity)

Reply:
I had them match the authconfig arguments that set them. Your suggestion is a much better approach. I will do that. Thank you.